CONTRIBUTING.md

```diff
@@ -0,0 +1,268 @@
+## Contributing to Ream
+
+Thanks for your interest in improving Ream!
+
+There are multiple opportunities to contribute at any level. It doesn't matter if you are just getting started with Rust
+or are the most weathered expert, we can use your help.
+
+**No contribution is too small and all contributions are valued.**
+
+This document will help you get started. **Do not let the document intimidate you**.
+It should be considered as a guide to help you navigate the process.
+
+The [Ream Telegram][ream-tg] is available for any concerns you may have that are not covered in this guide.
+
+If you contribute to this project, your contributions will be made to the project under the MIT license.
+
+### Code of Conduct
+
+The Ream project adheres to the [Rust Code of Conduct][rust-coc]. This code of conduct describes the _minimum_ behavior
+expected from all contributors.
+
+Instances of violations of the Code of Conduct can be reported by contacting the team
+at [kolby@reamlabs.com](mailto:kolby@reamlabs.com).
```

Comment:
I think this should be team's contact or @unnawut.

Reply:
I will put it to contact@reamlabs.com then for now

Reply:
Yep I can help handle non-technical issues so less distractions for @KolbyML.

Will loop everyone in if any report comes in anyway.

Reply:
> I will put it to [contact@reamlabs.com](mailto:contact@reamlabs.com) then for now

Using a shared contact email is great! Feel free to forward to my email address or give me non admin access to help monitor.

Reply:
> > I will put it to [contact@reamlabs.com](mailto:contact@reamlabs.com) then for now
> 
> Using a shared contact email is great! Feel free to forward to my email address or give me non admin access to help monitor.

I will dm you on telegram

Reply:
Emails are now forwarded